.gitignore

```diff
@@ -1,3 +1,7 @@
+telex.egg-info*
+*/telex.egg-info/*
+build/*
+dist/*
 *.pyc
-.venv/
```

Comment:
I don't think it hurts to keep these two around. I get the sense that the team doesn't necessary embrace `tox` so having virtualenv directories be ignored seems like a harmless idea.